packages/cli/lib/parse-run-one-options.ts

```diff
@@ -121,6 +116,14 @@ export function parseRunOneOptions(
   if ((!targets || !targets[target]) && projectIsNotSetExplicitly) return false;
   if (invalidTargetNames.indexOf(target) > -1) return false;
 
+  if (parsedArgs.configuration) {
+    configuration = parsedArgs.configuration;
+  } else if (parsedArgs.prod) {
+    configuration = 'production';
+  } else if (targets[target].defaultConfiguration) {
+    configuration = targets[target].defaultConfiguration;
```

Comment:
Thank you for supporting defaultConfiguration in Nx CLI! This should help React projects too!